server/app/uk/gov/ons/addressIndex/server/model/dao/AddressIndexElasticClientProvider.scala

```diff
@@ -73,11 +85,18 @@ class AddressIndexElasticClientProvider @Inject()
       requestConfigBuilder
     }
   }, (httpClientBuilder: HttpAsyncClientBuilder) => {
-    //      httpClientBuilder.setDefaultCredentialsProvider(provider)
+    if (basicAuth == "true") {
```

Comment:
The indentation seems off here, consider throwing it through IntelliJ's auto-formatter (Ctrl-Shift-L if I recall).